packages/app/app/stacks/empty.tsx

```diff
@@ -0,0 +1,22 @@
+import StackOfCoinsImage from "@/public/assets/images/empty-state-stacks-coins.svg";
+import { ButtonLink, HeadingText } from "@/ui";
+
+export const Empty = () => (
+  <div className="max-w-xl pt-32 mx-auto space-y-8">
+    <div className="flex flex-col items-center space-y-6 md:space-y-8">
+      <StackOfCoinsImage />
+      <div className="space-y-3 text-center">
+        <HeadingText size={2}>
+          {"Looks like you haven't created any stacks yet."}
```

Comment:
No need for this here, it can be simple text

Reply:
Typescript doesn't like all apostrophes ' .IMO it's easier like this than to use ampersand and stuff. But want to know your opinion.